README.md

```diff
@@ -31,23 +31,14 @@ En développement, copiez le fichier `.env.sample`, renommez-le `.env`, et mette
 
 Le serveur local de développement se lance au moyen des deux commandes suivantes :
 
-    & npm run db:build
     $ npm start
 
-Deux instances de développement sont alors accessibles :
+Trois instances de développement sont alors accessibles :
 
-- [localhost:3000](http://localhost:3000/) sert le frontend et le backend (API) ;
+- [localhost:8000](http://localhost:8000/) sert le backend django utilisé pour l'authentification, et sert aussi les fichiers statiques de elm
+- [localhost:8001](http://localhost:8001/) sert le frontend et le backend (API) ;
```

Comment:
C'est 8002 et 8001

Il faut aussi préciser que l'admin est accessible à `http://localhost:8002/admin/`

Reply:
En essayant de m'inscrire pour accéder à l'admin, j'ai ça 
![image](https://github.com/MTES-MCT/ecobalyse/assets/41547/601f144e-7710-443b-a2e0-f1bb7efbd144)

Il semble qu'il manque l'appel à la commande Django pour initialiser la base

Reply:
I lfaudra par ailleurs bien veiller à désactiver le mode debug de Django en production

Reply:
Running:

```
$ cd data/backend
$ ./manage.py migrate
Operations to perform:
  Apply all migrations: admin, auth, contenttypes, sessions
Running migrations:
  Applying contenttypes.0001_initial... OK
  Applying admin.0001_initial... OK
  Applying admin.0002_logentry_remove_auto_add... OK
  Applying admin.0003_logentry_add_action_flag_choices... OK
  Applying contenttypes.0002_remove_content_type_name... OK
  Applying auth.0001_initial... OK
  Applying auth.0002_alter_permission_name_max_length... OK
  Applying auth.0003_alter_user_email_max_length... OK
  Applying auth.0004_alter_user_username_opts... OK
  Applying auth.0005_alter_user_last_login_null... OK
  Applying auth.0006_require_contenttypes_0002... OK
  Applying auth.0007_alter_validators_add_error_messages... OK
  Applying auth.0008_alter_user_username_max_length... OK
  Applying auth.0009_alter_user_last_name_max_length... OK
  Applying auth.0010_alter_group_name_max_length... OK
  Applying auth.0011_update_proxy_permissions... OK
  Applying auth.0012_alter_user_first_name_max_length... OK
  Applying sessions.0001_initial... OK
```

seems to apply migrations, but I'm still getting the `no such table: authentication_ecobalyseuser` error…

Reply:
I've added `npm run auth:init` in the README which should make everything fall into place ;)

Reply:
The django debug mode can be (des)activated through an env var (so it can be activated or not on a scalingo review app deployment)